index.html

```diff
@@ -8,33 +8,179 @@
+            </div>
+            <div class="multi-post-stories-2-container">
+                <img src="images/multi-post-stories-2.png" alt="">
+                <p class="tonic">Multi-post stories</p>
+                <div class="canopy">
+                    <p>Canopy</p>
+                    <ul class="dev">
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="languages">
+                    <li>
+                        <button class="html">html</button>
+                    </li>
+                    <li>
+                        <button class="css">css</button>
+                    </li>
+                    <li >
+                        <button class="javascript">javascript</button>
+                    </li>
+                </ul>
+                <button class="btn">See Project</button>
+            </div> 
+        </section>
+        <section>
+            <div class="about-container">
+                <a class="about">About Myself</a>
+                <p class="details">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <a class="heading-1">LET’S CONNECT</a>
+                <ul class="icon-bar">
+                    <li>
+                        <img src="images/twitter.png" alt="">
+                    </li>
+                    <li>
+                        <img src="images/Linkedin icon.png" alt="">
+                    </li>
+                    <li>
+                        <img src="images/m.png" alt="">
+                    </li>
+                    <li>
+                        <img src="images/cat.png" alt="">
+                    </li>
+                    <li>
+                        <img src="images/hand.png" alt="">
+                    </li>
+                </ul>
+                <button class="btn-2">Get My Resume</button>
+                <p class="lang">Languages</p>
+                <ul class="lang-container">
+                    <li class="lang-item-js">
+                        <img src="images/js.png" alt="">
+                        <p>Javascript</p>
+                    </li>
+                    <li class="lang-item-html">
+                        <img src="images/html.png" alt="">
+                        <p>HTML</p>
+                    </li>
+                    <li class="lang-item-css">
+                        <img src="images/css.png" alt="">
+                        <p>CSS</p>
+                    </li>
+                </ul>
+                <p class="lang-p">Frameworks</p>
+                <hr>
+                <p class="lang-p">Skills</p>
```

Comment:
<img width="730" alt="3" src="https://user-images.githubusercontent.com/77843050/219653941-418051b7-3fae-4150-84f1-9aaae13f2580.png">

- [ ] Since I mentioned that all titles should be wrapped in a hx tag, in addition to that in this section, please add these arrow icons as in the Figma design, nothing else 😄

<img width="775" alt="2" src="https://user-images.githubusercontent.com/77843050/219654256-baf5eb4b-589b-4edb-8422-e82b8efee710.png">

- [ ] Work on aligning your framework as I have shown you in the image, I think you need to add more spaces on the left to make it breath.